[2024-07-18T15:41:04.238+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-07-18T15:41:04.266+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: run_fill_procedure.start_process manual__2024-07-18T15:41:03.366435+00:00 [queued]>
[2024-07-18T15:41:04.273+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: run_fill_procedure.start_process manual__2024-07-18T15:41:03.366435+00:00 [queued]>
[2024-07-18T15:41:04.274+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-07-18T15:41:04.286+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): start_process> on 2024-07-18 15:41:03.366435+00:00
[2024-07-18T15:41:04.292+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'run_fill_procedure', 'start_process', 'manual__2024-07-18T15:41:03.366435+00:00', '--job-id', '425', '--raw', '--subdir', 'DAGS_FOLDER/create_and_fill_proc.py', '--cfg-path', '/tmp/tmp0c3ww9x3']
[2024-07-18T15:41:04.294+0000] {standard_task_runner.py:91} INFO - Job 425: Subtask start_process
[2024-07-18T15:41:04.295+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=3544) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-07-18T15:41:04.296+0000] {standard_task_runner.py:63} INFO - Started process 3545 to run task
[2024-07-18T15:41:04.334+0000] {task_command.py:426} INFO - Running <TaskInstance: run_fill_procedure.start_process manual__2024-07-18T15:41:03.366435+00:00 [running]> on host 7c7e02e57379
[2024-07-18T15:41:04.401+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='run_fill_procedure' AIRFLOW_CTX_TASK_ID='start_process' AIRFLOW_CTX_EXECUTION_DATE='2024-07-18T15:41:03.366435+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-07-18T15:41:03.366435+00:00'
[2024-07-18T15:41:04.402+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-07-18T15:41:04.420+0000] {base.py:84} INFO - Using connection ID 'postgres_default' for task execution.
[2024-07-18T15:41:09.439+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-07-18T15:41:09.440+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-07-18T15:41:09.454+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=run_fill_procedure, task_id=start_process, run_id=manual__2024-07-18T15:41:03.366435+00:00, execution_date=20240718T154103, start_date=20240718T154104, end_date=20240718T154109
[2024-07-18T15:41:09.496+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-07-18T15:41:09.506+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/models/baseoperator.py:1297 AirflowProviderDeprecationWarning: Call to deprecated class PostgresOperator. (Please use `***.providers.common.sql.operators.sql.SQLExecuteQueryOperator`.Also, you can provide `hook_params={'schema': <database>}`.)
[2024-07-18T15:41:09.520+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-07-18T15:41:09.523+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
